### PR TITLE
build: use SFTP for launchpad uploads

### PR DESCRIPTION
**Pull request by fjl**
This switches PPA uploads to SFTP because Travis doesn't support FTP anymore. Keys are already configured on both Launchpad and Travis, so uploads should start working right after this PR is merged.

### CODE REVIEW

**Comment by karalabe** [comment]
PR failed https://travis-ci.org/ethereum/go-ethereum/jobs/492076005#L594

**Comment by holiman** [comment]
```
>>> dput -c /tmp/geth-build-509786312/dput.cf --no-upload-log ppa:ethereum/ethereum /tmp/geth-build-509786312/ethereum-swarm-unstable_0.3.11+build17470+trusty_source.changes
No host ppa found in config
```

**Comment by holiman** [comment]
Shouldn't it be
```
dput -c /tmp/geth-build-509786312/dput.cf --no-upload-log ppa.launchpad.net:ethereum/ethereum /tmp/geth-build-509786312/ethereum-swarm-unstable_0.3.11+build17470+trusty_source.changes`
```
?

**Comment by holiman** [comment]
Yes, you seem to be using `upload  = flag.String("upload", "", `Where to upload the source package (usually "ppa:ethereum/ethereum")`)` (the location of the upload) where the `host` should be

File: .travis.yml [hunk 1]
```diff
@@ -68,8 +68,10 @@ matrix:
             - debhelper
             - dput
             - fakeroot
+            - python-bzrlib
+            - python-paramiko
       script:
-        - go run build/ci.go debsrc -signer "Go Ethereum Linux Builder <geth-ci@ethereum.org>" -upload ppa:ethereum/ethereum
+        - go run build/ci.go debsrc -upload ppa:ethereum/ethereum -sftp-user geth-ci -signer "Go Ethereum Linux Builder <geth-ci@ethereum.org>"
```
Comment (holiman) [inline .travis.yml]: Since `fqdn` is defined in the config, it might work to use ` -upload ethereum/ethereum` ? The part before `:` is the host-part

**Pull request merged**